src/Directory.Build.props

```diff
@@ -21,6 +21,12 @@
     <NoWarn>$(NoWarn);1591</NoWarn>
   </PropertyGroup>
 
+  <!-- .NET source code analysis https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/overview -->
+  <PropertyGroup>
```

Comment:
As mentioned in the above link, this should be unnecessary.
> If your project targets .NET 5 or later, code analysis is enabled by default.

Reply:
May these lines trigger the GitHub Actions Check?
I've never seen these GitHub check annotations on this repo.

Reply:
The build/test one? That one is defined in https://github.com/bagetter/BaGetter/blob/main/.github/workflows/main.yml

Reply:
Enabling is not necessary, but setting the level to minimum is still a good idea so I kept that one.